osu.Game/Screens/OnlinePlay/Match/RoomSubScreen.cs

```diff
@@ -374,10 +375,16 @@ private void selectedItemChanged()
             }
         }
 
-        private void beatmapUpdated(BeatmapSetInfo set) => Schedule(updateWorkingBeatmap);
+        private void beatmapUpdated(BeatmapSetInfo set) => Schedule(UpdateWorkingBeatmap);
 
-        private void updateWorkingBeatmap()
+        protected virtual void UpdateWorkingBeatmap()
```

Comment:
Not a huge fan of this being `virtual`. Can we revert this to `private` and bind locally in the `MultiplayerMatchSubScreen` usage?

Reply:
See comment below.